lib/manageiq/api/common/filter.rb

```diff
@@ -23,30 +23,24 @@ def apply
               type = determine_string_attribute_type(attribute)
               send(type, k, v)
             else
-              errors.add("unsupported attribute type for: #{k}")
+              errors << "unsupported attribute type for: #{k}"
             end
           end
 
-          raise ManageIQ::API::Common::Filter::Error.new(:error_document => errors) unless errors.blank?
+          raise(ManageIQ::API::Common::Filter::Error, errors.join(", ")) unless errors.blank?
```

Comment:
I'm joining the errors with `", "` now unless you see some other way to add multiple errors @lindgrenj6 

Here's my diff in topo after upgrading:
```diff
       expect(response.parsed_body["errors"]).to eq(
         [
-          {"detail" => "unsupported attribute type for: mac_addresses", "status" => 400},
-          {"detail" => "found unpermitted parameter: bogus_attribute", "status" => 400}
+          {"detail" => "ManageIQ::API::Common::Filter::Error: unsupported attribute type for: mac_addresses, found unpermitted parameter: bogus_attribute", "status" => 400}
         ]
       )
```